solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -58,6 +62,67 @@ def apply_validation(obs_df, qfilter, handle_func):
         return validated_obs, counts
 
 
+def apply_fill(fx_data, missing_forecast, start, end, interval_length):
+    """
+    Apply fill procedure to the data from the start to end timestamps.
+
+    Parameters
+    ----------
+    fx_data : pandas.Series or pandas.DataFrame
+        Forecast data with pandas.DatetimeIndex.
+    missing_forecast : str
+        Indicates what process to use for handling missing forecasts.
+        Currently supports : 'drop', 'forward', and bool or numeric value.
+    start : pandas.Timestamp
+    end : pandas.Timestamp
+    interval_length : pandas.Timedelta
+
+    Returns
+    -------
+    pandas.Series or pandas.DataFrame
```

Comment:
```
filled: pandas.Series or pandas.DataFrame
    Forecast filled according to the specified logic
num : int
    Number of values filled or dropped
```

Reply:
Updated but changed from `num`, to `count`

Some of the docstrings have the name of the variable and others do not. The numpydocstring documentation says both are acceptable so whatever you are comfortable with. Do you want the names of the variables to match the variable in the code?

Reply:
my concern here was more that it was unclear that a tuple was returned